core/src/dird/ua_status.cc

```diff
@@ -372,39 +373,27 @@ static bool show_scheduled_preview(UaContext* ua,
                                    ScheduleResource* sched,
                                    PoolMem& overview,
                                    int* max_date_len,
-                                   struct tm tm,
                                    time_t time_to_check)
 {
-  int date_len, hour, mday, wday, month, wom, woy, yday;
-  bool is_last_week = false; /* Are we in the last week of a month? */
+  int date_len;
   char dt[MAX_TIME_LENGTH];
   time_t runtime;
   RunResource* run;
   PoolMem temp(PM_NAME);
 
-  hour = tm.tm_hour;
-  mday = tm.tm_mday - 1;
-  wday = tm.tm_wday;
-  month = tm.tm_mon;
-  wom = mday / 7;
-  woy = TmWoy(time_to_check); /* Get week of year */
-  yday = tm.tm_yday;          /* Get day of year */
-
-  is_last_week = IsDoyInLastWeek(tm.tm_year + 1900, yday);
+  RunHourValidator bt(time_to_check);
```

Comment:
the name `bt` seems to be a rather poor choice

Reply:
This was abbrevation the old name of the class. Will correct that.